releases/release-1.22/release-notes/maps/pr-102731-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 102731
+releasenote:
+  text: The `endpointslicemirroring` controller no longer mirrors the `last-applied-configuration`
```

Comment:
```suggestion
  text: The `EndpointSliceMirroring ` controller no longer mirrors the `last-applied-configuration`
```

Reply:
I will go with `EndpointSlice Mirroring Controller` as in https://discuss.kubernetes.io/t/last-week-in-kubernetes-development-week-ending-july-12-2020/11845